### PR TITLE
10 eye pressure explanation block scroll animation

### LINKED ISSUE #10: eye pressure explanation block
image

image

### PR DESCRIPTION
**Pull request by nfarmache-nc**
Fix #10 

Test URLs:
- Before: https://main--durysta--hlxsites.hlx.page
- After: https://10-eye-block--durysta--hlxsites.hlx.page/

### CODE REVIEW

**Comment by dragoss-nc** [comment]
Hello,

Some inconsistencies I found between dev (left) and live (right):

- Tablet: font size is different; also dot/pill position is a bit off, but not a big deal
image
- Mobile: text position is different, always near the corresponding pill, instead of below the eye; this is debatable, i like both approaches; from my side you could keep your vision.
image

File: blocks/scroll-animation/scroll-animation.css [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+  .scroll-animation .scroll-animation-body[data-index="3"] .scroll-animation-text {
+		top: 70%;
+		left: 82%;
+		width: 140px;
+  }
+}
+
+/* desktop break point */
+@media (min-width: 1200px) {
+  .scroll-animation .scroll-animation-body[data-index="1"] .scroll-animation-heading {
```
Comment (raeloh) [inline blocks/scroll-animation/scroll-animation.css:171]: This is a repeat of the styles in the `min-width: 768px` media query, so it's not necessary. However the text jumps dramatically between the 2 breakpoints.

File: blocks/scroll-animation/scroll-animation.css [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+  top: 9%;
+  left: 10%;
+}
+
+.scroll-animation .scroll-animation-body[data-index="2"] .scroll-animation-text {
+  top: 14%;
+  left: 35%;
+}
+
+.scroll-animation .scroll-animation-body[data-index="3"] .scroll-animation-heading {
```
Comment (raeloh) [inline blocks/scroll-animation/scroll-animation.css:113]: With this heading/text placement at the larger end of the <768px breakpoint, the spacing between the 2 heading and text is awkward 
Screen Shot 2023-07-10 at 09 01 42

**Comment by nfarmache-nc** [comment]
I pushed all the fixes for the differences from the original. I hope I covered everything.

**Comment by dragoss-nc** [comment]
On tablet + desktop I can see all texts, even if i don't scroll.

**Comment by nfarmache-nc** [comment]
Fixed small opacity bug

**Review by dragoss-nc — APPROVED** [review]

**Pull request merged**